tests/charts/api_tests.py

```diff
@@ -590,13 +679,19 @@ def test_update_chart(self):
         db.session.delete(model)
         db.session.commit()
 
+    @pytest.mark.usefixtures("load_birth_names_datasource")
     def test_update_chart_new_owner(self):
         """
         Chart API: Test update set new owner to current user
         """
+        ds_id = (
```

Comment:
It looks like this lookup could be moved to a shared setup function

Reply:
I just moved it to a function to reduce repetitions. Moving it to `setUp` would cause need to apply fixture to all tests in class.